.github/SECURITY.md

```diff
@@ -1,61 +1,50 @@
-    under the License.
-
-Security Model
---------------
-
-In the Airflow security model, the system administrators are fully trusted.
-They are the only ones who can upload new DAGs, which gives them the ability
-to execute any code on the server.
-
-Authenticated web interface and API users with Admin/Op permissions are trusted,
-but to a lesser extent: they can configure the DAGs which gives them some control,
-but not arbitrary code execution.
-
-Authenticated Web interface and API users with 'regular' permissions are trusted
-to the point where they can impact resource consumption and pause/unpause configured DAGs,
-but not otherwise influence their functionality.
-
-Reporting Vulnerabilities
--------------------------
+## Reporting Vulnerabilities
 
 **⚠️ Please do not file GitHub issues for security vulnerabilities as they are public! ⚠️**
 
 The Apache Software Foundation takes security issues very seriously. Apache
 Airflow specifically offers security features and is responsive to issues
 around its features. If you have any concern around Airflow Security or believe
 you have uncovered a vulnerability, we suggest that you get in touch via the
-e-mail address security@airflow.apache.org. In the message, try to provide a
-description of the issue and ideally a way of reproducing it. The security team
-will get back to you after assessing the description.
+e-mail address [security@airflow.apache.org](mailto:security@airflow.apache.org).
+In the message, try to provide a description of the issue and ideally a way of
+reproducing it. The security team will get back to you after assessing the report.
 
 Note that this security address should be used only for undisclosed
 vulnerabilities. Dealing with fixed issues or general questions on how to use
 the security features should be handled regularly via the user and the dev
 lists. Please report any security problems to the project security address
 before disclosing it publicly.
 
-The `ASF Security team's page <https://www.apache.org/security/>`_ describes
-how vulnerability reports are handled, and includes PGP keys if you wish to use
-that.
+Before reporting vulnerabilities, please make sure to read and understand the
+[security model](https://airflow.apache.org/docs/apache-airflow/stable/security/) of Airflow, because
+some of the potential security vulnerabilities that are valid for projects that are publicly accessible
+from the Internet, are not valid for Airflow. Airflow is not designed to be used by untrusted users, and some
+trusted users are trusted enough to do a variety of operations that could be considered as vulnerabilities
+in other products/circumstances. This is the reason why some potential security vulnerabilities do not
+apply to Airflow, or have different severity than generic scoring systems (for example `CVSS`)
+calculation suggests.
 
+The [ASF Security team's page](https://www.apache.org/security/>) describes
```

Comment:
```suggestion
The [ASF Security team's page](https://www.apache.org/security/) describes
```

oops sorry missed reviewing this earlier. Looks like we have extra `>`, had just clicked on the link but forgot to verify earlier that the new tab opened the link correctly or not. Sorry for the multiple rounds of review comments.

Reply:
No worries at all :)